modules/integration/pom.xml

```diff
@@ -96,6 +96,11 @@
       <artifactId>zookeeper</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Correction: the modules/cluster/pom.xml does specify logback explicitly (though, not as `<optional>true</optional>`, perhaps because it is a hard requirement, due to Twill).